kafka-statefulsets/scripts/kafka_run.sh

```diff
@@ -20,6 +20,6 @@ echo "LOG_DIR=$LOG_DIR"
 # starting Kafka server with final configuration
 exec $KAFKA_HOME/bin/kafka-server-start.sh $KAFKA_HOME/config/server.properties \
 --override broker.id=$KAFKA_BROKER_ID \
---override advertised.host.name=$(hostname -I) \
+--override advertised.host.name=$(hostname -f) \
```

Comment:
`advertised.host.name` is deprecated, replaced by `advertised.listeners`.

Reply:
I know. But I didn't wanted to rewrite more than necessary - I just wanted to fix what seems like a minor bug. We have another issues opened to change how Kafka is configured such as #27 or #40. I suggest to leave advertised.listener for them. WDYT?

Reply:
OK, let's leave it for now.